src/extensions/renderer/canvas/drawing-images.js

```diff
@@ -23,9 +23,11 @@ CRp.drawInscribedImage = function( context, img, node, index ){
   var nodeY = pos.y;
   var fit = getIndexedStyle( node, 'background-fit', 'value', index );
   var repeat = getIndexedStyle( node, 'background-repeat', 'value', index );
-  var paddingX2 = node.pstyle( 'padding' ).pfValue * 2;
-  var nodeW = node.width() + ( getIndexedStyle( node, 'background-width-relative-to', 'value', index ) === 'inner' ? 0 : paddingX2 );
-  var nodeH = node.height() + ( getIndexedStyle( node, 'background-height-relative-to', 'value', index ) === 'inner' ? 0 : paddingX2 );
+  var nodeW = node.width();
+  var nodeH = node.height();
+  var paddingX2 = util.computePaddingValues(nodeW, nodeH, node.pstyle( 'padding' ), node.pstyle( 'padding-relative-to' ).value ) * 2;
+  var nodeTW = nodeW + ( getIndexedStyle( node, 'background-width-relative-to', 'value', index ) === 'inner' ? 0 : paddingX2 );
```

Comment:
Updating the compound bounds should update all the auto values.

I think percent padding is only supposed to work for parent nodes.  There's not much of a usecase of using percent paddings for non-parent nodes, because they're not dynamic like parents.  The point of having functions like `width()` and `padding()` is that you don't want to repeat style rules over several files --- a change just needs to update one function.

If it's not documented already, we should mention in the docs that percent padding is for compounds only.